scripts/zones/Southern_San_dOria/npcs/Cletae.lua

```diff
@@ -6,30 +6,28 @@
 -----------------------------------
 local ID = require("scripts/zones/Southern_San_dOria/IDs")
 require("scripts/globals/shop")
-require("scripts/globals/crafting")
 require("scripts/globals/settings")
 require("scripts/globals/status")
+require("scripts/globals/npc_util")
 -----------------------------------
 
-function onTrade(player,npc,trade)
-    if (FlyerForRegine == 1) then
-        count = trade:getItemCount()
-        MagicFlyer = trade:hasItemQty(532,1)
-        if (MagicFlyer == true and count == 1) then
+function onTrade(player, npc, trade)
+    if FlyerForRegine == 1 then
```

Comment:
Undefined variable: FlyerForRegine (this issue existed in the code before you edited it)

The Flyers for Regine quest got a major update on retail at some point -- see recent issue and my related PRs, that implement more retail-accurate behavior.  Now, you don't need to randomly trade flyers to everyone.  Instead, the 15 NPCs who will accept flyers give you a message when you approach them.  I'm guessing that because of this on retail, NPCs who -don't- accept flyers may no longer give refusal messages, so this code block may not be needed any longer.  However, retail confirmation is needed.

Reply:
that's a valid point, i'm going to ask one of the testers to check out if this behavior is still needed, if so a new PR will be posted to focus solely on it. :cat:  thanks for bringing this up @wrenffxi

Reply:
this will still throw and error in the meantime. may as well just comment out or remove if you aren't going to define the variable you are using till a new pr.

Reply:
@TeoTwawki  added the var and a also a note.